euphoria-operator-testkit/src/main/java/cz/seznam/euphoria/operator/test/ReduceByKeyTest.java

```diff
@@ -126,119 +126,119 @@ public void validate(Partitions<Pair<Integer, HashSet<Integer>>> partitions) {
+//        }
+//
+//        assertEquals(2, byKey.size());
+//
+//        assertNotNull(byKey.get(0));
+//        assertEquals(3, byKey.get(0).size());
+//        assertEquals(Arrays.asList(2, 6, 12), byKey.get(0));
+//
+//        // ~ cannot make any assumption about the order of the elements in the windows
+//        // (on batch) since we have no event-time order for the test data
+//        assertNotNull(byKey.get(1));
+//        assertEquals(
+//            Sets.newHashSet(1, 4, 9, 7, 16),
+//            byKey.get(1).stream().collect(Collectors.toSet()));
+//      }
+//    });
+//  }
+//
+//
+//  @Test
+//  public void testEventTime() {
+//    execute(new AbstractTestCase<Pair<Integer, Long>, Pair<Integer, Long>>() {
+//
+//      @Override
+//      protected Dataset<Pair<Integer, Long>> getOutput(Dataset<Pair<Integer, Long>> input) {
+//        input = AssignEventTime.of(input).using(Pair::getSecond).output();
+//        return ReduceByKey.of(input)
+//            .keyBy(Pair::getFirst)
+//            .valueBy(e -> 1L)
+//            .combineBy(Sums.ofLongs())
+//            .setPartitioner(e -> e % 2)
+//            .windowBy(Time.of(Duration.ofSeconds(1)))
+//            .output();
+//      }
+//
+//      @Override
+//      protected Partitions<Pair<Integer, Long>> getInput() {
+//        return Partitions
+//            .add(Pair.of(1, 300L), Pair.of(2, 600L), Pair.of(3, 900L),
+//                Pair.of(2, 1300L), Pair.of(3, 1600L), Pair.of(1, 1900L),
+//                Pair.of(3, 2300L), Pair.of(2, 2600L), Pair.of(1, 2900L),
+//                Pair.of(2, 3300L))
+//            .add(Pair.of(2, 300L), Pair.of(4, 600L), Pair.of(3, 900L),
+//                Pair.of(4, 1300L), Pair.of(2, 1600L), Pair.of(3, 1900L),
+//                Pair.of(4, 2300L), Pair.of(1, 2600L), Pair.of(3, 2900L),
+//                Pair.of(4, 3300L), Pair.of(3, 3600L))
+//            .build();
+//      }
+//
+//      @Override
+//      public int getNumOutputPartitions() {
+//        return 2;
+//      }
+//
+//      @Override
+//      public void validate(Partitions<Pair<Integer, Long>> partitions) {
+//        assertEquals(2, partitions.size());
+//        List<Pair<Integer, Long>> first = partitions.get(0);
+//        assertUnorderedEquals(Arrays.asList(
+//            Pair.of(2, 2L), Pair.of(4, 1L),  // first window
+//            Pair.of(2, 2L), Pair.of(4, 1L),  // second window
+//            Pair.of(2, 1L), Pair.of(4, 1L),  // third window
+//            Pair.of(2, 1L), Pair.of(4, 1L)), // fourth window
+//            first);
+//        List<Pair<Integer, Long>> second = partitions.get(1);
+//        assertUnorderedEquals(Arrays.asList(
+//            Pair.of(1, 1L), Pair.of(3, 2L),  // first window
+//            Pair.of(1, 1L), Pair.of(3, 2L),  // second window
+//            Pair.of(1, 2L), Pair.of(3, 2L),  // third window
+//            Pair.of(3, 1L)),                 // fourth window
+//            second);
+//      }
+//    });
+//  }
```

Comment:
Why do we comment this out? If it is obsolete for some reason, shouldn't we just drop it?